benchmarks/tables/bpdn-constr-table.jl

```diff
@@ -6,18 +6,20 @@ compound = 1
 model, nls_model, sol = bpdn_model(compound, bounds = true)
 
 f = LSR1Model(model)
-λ = norm(grad(model, zeros(model.meta.nvar)), Inf) / 10
+λ = 1.0e-1 # norm(grad(model, zeros(model.meta.nvar)), Inf) / 10
```

Comment:
Are the results very different if you  leave the default lambda ?

Reply:
Not that much, I updated the tables and the benchmarks in the article.